src/editors/data/redux/thunkActions/video.js

```diff
@@ -24,12 +24,20 @@ export const uploadTranscript = ({ language, filename, file }) => (dispatch, get
     language: lang,
     videoId,
     transcript: file,
-    onSuccess: () => dispatch(actions.video.updateField({
-      transcripts: {
-        ...transcripts,
-        [lang]: { filename },
-      },
-    })),
+    onSuccess: (response) => {
+      dispatch(actions.video.updateField({
+        transcripts: {
+          ...transcripts,
+          [lang]: { filename },
+        },
+      }));
+      if (selectors.video.videoId(state) === '') {
```

Comment:
@rayzhou-bit this is where that code would occur, you are looking at the requests code I believe.

Reply:
Oh derp. I missed this